File: ocel/examples/import_jsonocel_export_jsonocel.py
import ocel


def execute_script():
    print("validated input", ocel.validate("../logs/minimal.jsonocel", "../schemas/schema.json"))
    log = ocel.import_log("../logs/minimal.jsonocel")
    ocel.export_log(log, "log1.jsonocel")
    print("validated output", ocel.validate("log1.jsonocel", "../schemas/schema.json"))
    ocel.import_log("log1.jsonocel")


if __name__ == "__main__":
    execute_script()

File: ocel/src/main/java/it/unibz/inf/pm/ocel/examples/ImportJsonocelExportJsonocel.java
package it.unibz.inf.pm.ocel.examples;

import it.unibz.inf.pm.ocel.OcelInitial;
import org.dom4j.DocumentException;

public class ImportJsonocelExportJsonocel {

    public static void main(String[] args) throws Exception {
        try {
            System.out.print("validated input:");
            OcelInitial.validate("ocel/logs/minimal.jsonocel", "ocel/schemas/schema.json");
            Object log = OcelInitial.import_log("ocel/logs/minimal.jsonocel");
            OcelInitial.export_log(log,"ocel/examples/log1.jsonocel");

            System.out.print("\nvalidated output:");
            OcelInitial.validate("ocel/examples/log1.jsonocel", "ocel/schemas/schema.json");
        } catch (DocumentException e) {
            e.printStackTrace();
        }
    }
}
